admin.py

```diff
@@ -381,7 +381,10 @@ def post(self, path, feature_id=None):
       common.handle_401(self.request, self.response, Exception)
       return
 
+    spec_repo_url = self.__FullQualifyLink('spec_repo_url')
     spec_link = self.__FullQualifyLink('spec_link')
+    security_review_url = self.__FullQualifyLink('security_review_url')
+    privacy_review_url = self.__FullQualifyLink('privacy_review_url')
```

Comment:
The PING process is a bit weird.  (This is, I think, supposed to be external privacy review; internal privacy review will happen with launch bug.) I think this *will* eventually be a "URL to privacy review request", but I've got an ask out for clarification.  Until then, let's just hide this?